backend/cache_utils.py

```
@@ -86,3 +86,37 @@ def invalidate_tab_feeds_cache(tab_id, invalidate_tabs=True):
+    # Cast to list to prevent generator exhaustion
+    tab_ids_list = list(tab_ids)
+    if not tab_ids_list:
+        return
+
+    version_keys = [get_tab_version_key(tab_id) for tab_id in tab_ids_list]
+
+    # cache.get_many accepts multiple positional arguments
+    versions = cache.get_many(*version_keys)
+
+    updates = {}
+    for key, version in zip(version_keys, versions):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)